wsmaster/codenvy-hosted-api-license/src/main/java/com/codenvy/api/license/server/license/CodenvyLicenseManager.java

```diff
@@ -40,23 +43,31 @@
 
     private final CodenvyLicenseFactory licenseFactory;
     private final Path                  licenseFile;
+    private final AdminUserDao          adminUserDao;
+    private final SwarmDockerConnector  dockerConnector;
 
     @Inject
-    public CodenvyLicenseManager(@Named("license-manager.license-file") String licenseFile, CodenvyLicenseFactory licenseFactory) {
+    public CodenvyLicenseManager(@Named("license-manager.license-file") String licenseFile,
+                                 CodenvyLicenseFactory licenseFactory,
+                                 AdminUserDao adminUserDao,
+                                 SwarmDockerConnector dockerConnector) {
         this.licenseFactory = licenseFactory;
         this.licenseFile = Paths.get(licenseFile);
+        this.adminUserDao = adminUserDao;
+        this.dockerConnector = dockerConnector;
     }
 
     /**
      * Stores valid Codenvy license into the storage.
      *
      * @throws NullPointerException
-     *         if {@code codenvyLicense} is null
+     *         if {@code licenseStr} is null
```

Comment:
licenseText and licenseContent hardly ever helps us to defer the values for sure.